qos-monitor/src/main/java/eu/arrowhead/core/qos/QosMonitorConstants.java

```diff
@@ -0,0 +1,43 @@
+package eu.arrowhead.core.qos;
+
+public class QosMonitorConstants {
+
+	//=================================================================================================
+	// members
+	public static final String EVENT_QUEUE = "EventQueue";
+	public static final String EVENT_BUFFER = "EventBuffer";
+	public static final String EVENT_COLLECTOR = "EventCollector";
+
+	public static final int EVENT_LIST_SIZE = 4;
+
+	public static final String PROCESS_ID_KEY = "processID";
+
+	public static final String RECEIVED_MONITORING_REQUEST_EVENT_PAYLOAD_SCHEMA = "RECEIVED_MONITORING_REQUEST";
```

Comment:
still used the constant string payload? If I remember correctly, we agreed that we use empty list as payload where there is not payload.

Reply:
fixed